src/components/Input/components/Switch/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import InputBase from '../InputBase';
+import { Props } from './types';
+
+
+function Switch({
+    className,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const switchClasses = clsx(
+        'brew-Switch',
```

Comment:
The base classname should be `brew-Input--Switch` since it's an input component

Reply:
Based on BEM though, that is saying that `Switch` is a modifier of `Input`, which it's really not considering it's a truly separate component. We also won't have another `Switch` component, so unless we want to take the approach that we use parent categories in our BEM bases throughout, I think it's fine as is. 

If we do change it, I would advocated `brew-Input-Switch` instead of the modifier syntax.